datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -58,6 +84,16 @@ def create_codeowners_map():
 def codeowners(ctx):
     """Validate that every integration has an entry in the `CODEOWNERS` file."""
 
+    failed_integrations = validate_logs_assets_codeowners()
+    if failed_integrations:
+        for integration in failed_integrations:
+            echo_failure(f"/{integration}/assets/logs/ is not owned by {LOGS_TEAM}")
+        abort()
+    else:
+        echo_success("All integrations have valid logs codeowners.")
+
+    codeowner_map = create_codeowners_map()
+
     has_failed = False
```

Comment:
and thus move this to the top 🙂

Reply:
Do you mean to the bottom ? Like, this commit: [cc7deb8](https://github.com/DataDog/integrations-core/pull/17185/commits/cc7deb845bd5902e36743f43ae8786d927757108)

Reply:
I meant `has_failed = False`, so it works if you put your code at the bottom 😄

Reply:
aaaah, haha. Makes sense, I did it even if I had not fully understood your comment :P